src/views/Restoration.vue

```diff
@@ -225,10 +231,17 @@ export default class Restoration extends Vue {
       // set Applicant info in entitySnapshot
       const stateFiling = entitySnapshot.businessInfo.stateFiling
       const filing = stateFiling && await LegalServices.fetchFiling(stateFiling)
-      entitySnapshot.orgPersons = filing.restoration.parties as OrgPersonIF
-      entitySnapshot.orgPersons.forEach(o => {
-        o.officer.id = uuidv4()
-      })
+      if (!filing) {
+        throw new Error(`Invalid fetched stateFiling = ${this.getBusinessId}`)
+      } else {
```

Comment:
You don't need the else since the thrown error will return from this code block.